scss/_reboot.scss

```diff
@@ -373,147 +381,159 @@ select {
 // 1. Prevent a WebKit bug where (2) destroys native `audio` and `video`
 //    controls in Android 4.
 // 2. Correct the inability to style clickable types in iOS and Safari.
+// 3. Opinionated: add "hand" cursor to non-disabled button elements.
+
 button,
 [type="button"], // 1
 [type="reset"],
 [type="submit"] {
   -webkit-appearance: button; // 2
-}
 
-// Opinionated: add "hand" cursor to non-disabled button elements.
-@if $enable-pointer-cursor-for-buttons {
-  button,
-  [type="button"],
-  [type="reset"],
-  [type="submit"] {
+  @if $enable-pointer-cursor-for-buttons {
     &:not(:disabled) {
-      cursor: pointer;
+      cursor: pointer; // 3
     }
   }
 }
 
+
+// Fix height of inputs with a type of datetime-local, date, month, week, or time
+// See https://github.com/twbs/bootstrap/issues/18842
+
+::-webkit-datetime-edit {
+  line-height: 1;
+}
```

Comment:
I don't understand this point. Could you teach me more in detail about this?

Reply:
See https://codepen.io/MartijnCuppens/pen/MMywaL

I did this so we might might switch from `height` to `min-height` for form controls, so we don't have height issues anymore when the form control is within a flex container. We could also remove this:

https://github.com/twbs/bootstrap/blob/ceffc3a5fd7af7609150048f0651f025ef19672c/scss/_forms.scss#L153-L163

Was planning to investigate that when this is merged, I might also just add this in another PR

Reply:
Looks good on the desktop and Android browsers. but I found the #23307 is still reproduced (but not relative on this PR).

Reply:
I removed this in favor of https://github.com/twbs/bootstrap/blob/cda30cc9950949e9e85320089551b83380f1f22a/scss/_reboot.scss#L516-L522

The line height fix didn't seem to work in all scenarios